src/libcommonserver/io/iohelper.cpp

```diff
@@ -276,6 +280,46 @@ bool IoHelper::_checkIfIsHiddenFile(const SyncPath &path, bool &isHidden, IoErro
 
     return true;
 }
+
+bool IoHelper::getRights(const SyncPath &path, bool &read, bool &write, bool &exec, IoError &ioError) noexcept {
+    read = false;
+    write = false;
+    exec = false;
+
```

Comment:
It might be good to initialize `ioError` here, just for our peace of mind.

Reply:
OK